skunkworks_crow/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,103 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+
+<!-- Common dimensions for all layout files-->
+
+<!--Layout dimens-->
+    <dimen name="linear_layout_padding">10dp</dimen>
+    <dimen name="root_layout_padding">16dp</dimen>
+    <dimen name="root_layout_padding_top">8dp</dimen>
+    <dimen name="root_layout_padding_bottom">8dp</dimen>
+
+    <dimen name="relative_layout_padding">8dp</dimen>
+
+    <dimen name="webView_root_layout_margin">0dp</dimen>
+    <dimen name="webView_root_layout_padding">0dp</dimen>
+
+
+    <!--image view dimens-->
+    <dimen name="image_view_margin">10dp</dimen>
+    <dimen name="image_view_height">144dp</dimen>
+    <dimen name="image_view_width">144dp</dimen>
+    <dimen name="image_view_padding">12dp</dimen>
+
+    <dimen name="image_view_padding_start">6dp</dimen>
```

Comment:
All the ImageView padding are same throught the app, so lets create only one dimens resource which will server the purpose of all four.